ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -1685,6 +1685,16 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-automated-release
     workflow: cucushift-installer-rehearse-vsphere-upi-disconnecting-network
+- as: aws-ipi-f7-netobserv
+  cron: 0 0 1,8,17,26 * *
```

Comment:
Currently, those jobs added in this PR runs on the same day/time, which is more likely to met quota issues.
Would you mind to use this small [shell script](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh) to generate a different cron entry for them to spread out the jobs ?

Reply:
done, updated